python/lsst/dax/imgserv/imageREST_v1.py

```diff
@@ -154,19 +158,37 @@ def _getimage(_req, db_id):
         request the request object
         db  image database string
     """
-    ds = _req.args.get("ds")
+    if _req.method == "POST":
+        # check  mimetype
+        mime_type =_req.headers["Content-Type"]
```

Comment:
I think you should check before here if the input is actually a form too, in which case, treat the POST request the same as the GET request in line 178. I think that should work just fine. We can add a ticket to accept this improvement later too.

Reply:
I've implemented support for form post here as well, resulting in much simpler the logic flow.